01week/resume/style.css

```diff
@@ -0,0 +1,44 @@
+#name {
```

Comment:
I think you used atom-beautify here because circleci is saying that every line here is indented two additional spaces. I found atom-beautify to be unreliable, so I just uninstalled it.